lib/src/main/java/dataframe/transformation/string/Mask.java

```diff
@@ -34,14 +34,20 @@ public Mask(int characters, char maskCharacter, String direction) {
 
     @Override
     protected Object applyOnString(String s) {
-        if(characters <= 0) {
-            return String.valueOf(maskCharacter).repeat(s.length());
+        if(s.trim().isEmpty()){
+            return s;
+        } else{
+            String maskString = String.valueOf(this.maskCharacter).repeat(this.characters);
+            switch (this.direction) {
+                case LEFT -> {
+                    return maskString + s.substring(Math.min(this.characters, s.length()));
+                }
+                case RIGHT -> {
+                    return s.substring(0, Math.max(0, s.length() - this.characters)) + maskString;
+                }
```

Comment:
use the new switch pattern. `return` will come before `switch` and inside case statement we return with `yield`
Example:
```
return switch(this.direction) {
    case LEFT -> {
        yield result
    }
}
```